src/pybind/mgr/volumes/fs/vol_spec.py

```diff
@@ -9,6 +15,8 @@ class VolSpec(object):
     DEFAULT_NS_PREFIX = "fsvolumens_"
     # default mode for subvol prefix and group
     DEFAULT_MODE = 0o755
+    # internal directories
+    INTERNAL_DIRS = [Group.NO_GROUP_NAME, Index.GROUP_NAME, Trash.GROUP_NAME, SubvolumeBase.LEGACY_CONF_DIR]
```

Comment:
The main PR doesn't have this. I think this is dependent on https://github.com/ceph/ceph/pull/47474 which is merged in quicny. Please rebase this PR on top of it.

Reply:
Sure. As https://github.com/ceph/ceph/pull/47474 merged, I need to rebase.